src/spyglass/decoding/v0/clusterless.py

```diff
@@ -199,8 +204,8 @@ def make(self, key):
 
             marks = np.concatenate(
                 [
-                    UnitMarks._get_peak_amplitude(
-                        waveform=waveform_extractor.get_waveforms(unit_id),
+                    _get_peak_amplitude(
+                        waveform_extractor=waveform_extractor,
                         peak_sign=peak_sign,
```

Comment:
I moved it to `decoding/utils.py` [here](https://github.com/CBroz1/spyglass/blob/dpe/src/spyglass/decoding/utils.py#L28) to let both versions pass the extractor. Is there something about this parent func that requires otherwise?

Reply:
Oh no sorry, I mean this call to `_get_peak_amplitude` needs to pass a value for `unit_id`

```suggestion
                        peak_sign=peak_sign,
                        unit_id=unit_id,
```